### ISSUE TITLE
Testing Push Notification

### ISSUE THREAD

**Issue by entrustca** [vincentmorneau/apex-pwa#15]
Good day,

I setup everything in the guide for Push Notification , I used the sample pl/sql script:

declare
  l_rest_return clob;
begin
  l_rest_return := apex_web_service.make_rest_request(
    p_url => 'https://localhost:3050/push' -- My custom REST endpoint (1)
    , p_http_method => 'GET'
    , p_parm_name => apex_util.string_to_table('title:body') -- Notification structure (2)
    , p_parm_value => apex_util.string_to_table('Hey you:Go back to APEX now!') -- Notification Content (3)
  );
end;

And get the following error:

Error report -
ORA-29273: HTTP request failed
ORA-06512: at "SYS.UTL_HTTP", line 1130
ORA-29259: end-of-input reached
ORA-06512: at "APEX_180200.WWV_FLOW_WEB_SERVICES", line 1017
ORA-06512: at "APEX_180200.WWV_FLOW_WEB_SERVICES", line 1523
ORA-06512: at "APEX_180200.WWV_FLOW_WEBSERVICES_API", line 369
ORA-06512: at line 4
29273. 00000 -  "HTTP request failed"
*Cause:    The UTL_HTTP package failed to execute the HTTP request.
*Action:   Use get_detailed_sqlerrm to check the detailed error message.
           Fix the error and retry the HTTP request.

Please help me to solve this issue.

Regards

**Comment by vincentmorneau** [comment]
Have you set up and launched your node service?

**Comment by entrustca** [comment]
Yes I did, if I put POST instead of GET it finished normal but I don't see any notification. I also setup the server in my laptop but I received and ORA no TNS Listener message. Please let me know what else I can do on my laptop and server.

Thanks

**Comment by entrustca** [comment]
This is the error when I run it using my laptop with nodejs:

ORA-29273: HTTP request failed
ORA-06512: at "SYS.UTL_HTTP", line 1722
ORA-12541: TNS:no listener
ORA-06512: at line 1
29273. 00000 -  "HTTP request failed"
*Cause:    The UTL_HTTP package failed to execute the HTTP request.
*Action:   Use get_detailed_sqlerrm to check the detailed error message.
           Fix the error and retry the HTTP request.

**Comment by entrustca** [comment]
I setup a local machine with centos and the same error continues:

SQL> declare
  l_rest_return clob;
begin
  l_rest_return := apex_web_service.make_rest_request(
    p_url => 'http://localhost:3050/push' -- My custom REST endpoint (1)
    , p_http_method => 'GET'
    , p_parm_name => apex_util.string_to_table('title:body') -- Notification structure (2)
    , p_parm_value => apex_util.string_to_table('Hey you:Go back to APEX now!') -- Notification Content (3)
  );
end;  2    3    4    5    6    7    8    9   10  
 11  /
declare
*
ERROR at line 1:
ORA-29273: HTTP request failed
ORA-06512: at "SYS.UTL_HTTP", line 1369
ORA-29259: end-of-input reached
ORA-06512: at "APEX_180100.WWV_FLOW_WEB_SERVICES", line 1011
ORA-06512: at "APEX_180100.WWV_FLOW_WEB_SERVICES", line 1517
ORA-06512: at "APEX_180100.WWV_FLOW_WEBSERVICES_API", line 369
ORA-06512: at line 4

Please let me know what to look.

Regards

**Comment by APXUSER** [comment]
Dears,
I have the same prob. Did you save it ?
Database 11g Release 2. APEX 18.1

Thanks in advance.

ERROR at line 1:
ORA-29273: HTTP request failed
ORA-06512: at "SYS.UTL_HTTP", line 1369
ORA-29259: end-of-input reached
ORA-06512: at "APEX_180100.WWV_FLOW_WEB_SERVICES", line 1011
ORA-06512: at "APEX_180100.WWV_FLOW_WEB_SERVICES", line 1517
ORA-06512: at "APEX_180100.WWV_FLOW_WEBSERVICES_API", line 369
ORA-06512: at line 4

**Comment by entrustca** [comment]
Hello I set everything as the tutorial but it looks that it does not connect to the firebase database I put the firebase json but it does not connect do I need to do an extra setup in firebase? Thanks

**Comment by vincentmorneau** [comment]
Please post your node server code. Can't help you without this information...
